pkg/config/toenv.go

```diff
@@ -66,13 +64,22 @@ func mapToEnv(prefix string, data map[string]interface{}) []string {
 		keyName = strings.ReplaceAll(keyName, "-", "_")
 		// Apply overrides needed to convert between configs types
 		keyName = envOverrides(keyName)
-		if keyName == "SKIP" {
-			continue
-		}
 
 		if data, ok := v.(map[string]interface{}); ok {
 			subResult := mapToEnv(keyName+"_", data)
 			result = append(result, subResult...)
+		} else if slice, ok := v.([]interface{}); ok {
```

Comment:
I am not very happy with this logic, but passing arrays as environment variables essentially sucks. 🤷🏽‍♂️ 

A comma separated value format is supported out of the box in `elemental-cli` (see rancher/elemental-cli#287) thanks to viper/cobra. There is no escaping mechanism as I consider an insane practice naming files including characters such as commas...